### PR TITLE
refactor(pipelines): convert manual judgment options to react

### PR DESCRIPTION
**Pull request by anotherchrisberry**
No functionality change, just converting to React.

I am already a React expert so no need to look at this closely unless you want to see how the pros do it @jrsquared @christopherthielen

### CODE REVIEW

File: app/scripts/modules/core/pipeline/config/stages/manualJudgment/ManualJudgmentApproval.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    // do not update the submitting state - the reload of the executions will clear it out; otherwise,
+    // there is a flash on the screen when we go from submitting to not submitting to the buttons not being there.
+    this.props.application.getDataSource('executions').refresh();
+  }
+
+  private judgmentOptionSelected(option: Select.Option): void {
+    this.setState({judgmentInput: {value: option.value as string}});
+  }
+
+  public judgmentFailure(): void {
```
Comment (jrsquared) [inline app/scripts/modules/core/pipeline/config/stages/manualJudgment/ManualJudgmentApproval.tsx]: why is this one public and the rest private?

File: app/scripts/modules/core/pipeline/config/stages/manualJudgment/ManualJudgmentApproval.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      <div>
+        { showOptions && (
+          <div>
+            { options.length > 0 && (
+              <div className="form-group col-md-12">
+                <p><b>Judgment Input</b></p>
+                <Select options={options}
+                        clearable={false}
+                        value={this.state.judgmentInput.value}
+                        onChange={(e: string) => this.judgmentOptionSelected(e)}/>
```
Comment (jrsquared) [inline app/scripts/modules/core/pipeline/config/stages/manualJudgment/ManualJudgmentApproval.tsx]: `e` is usually short for event, in this case it is actually a `Select.Option` so maybe `option`?

**Comment by jrsquared** [comment]
Well, you know, except fix the tests.

**Comment by christopherthielen** [comment]
+180 −84
progress!

File: app/scripts/modules/core/pipeline/config/stages/manualJudgment/ManualJudgmentApproval.tsx (hunk 1 above)
Comment (anotherchrisberry) [inline app/scripts/modules/core/pipeline/config/stages/manualJudgment/ManualJudgmentApproval.tsx]: habit from Angular component controller - thanks for catching

File: app/scripts/modules/core/pipeline/config/stages/manualJudgment/ManualJudgmentApproval.tsx (hunk 2 above)
Comment (anotherchrisberry) [inline app/scripts/modules/core/pipeline/config/stages/manualJudgment/ManualJudgmentApproval.tsx]: ah, good point, will fix

**Comment by anotherchrisberry** [comment]
@christopherthielen I can make those number worse, give me a sec

**Pull request merged**